docs/source/configuration/tracing.mdx

```diff
@@ -125,8 +129,7 @@ telemetry:
 
 ## Trace ID
 
-> This is part of an experimental feature, it means any time until it's stabilized (without the prefix `experimental_`) we might change the configuration shape or adding/removing features.
-> If you want to give feedback or participate in that feature feel free to join [this discussion on GitHub](https://github.com/apollographql/router/discussions/2147).
+<ExperimentalFeature appendText="You can also give feedback in the [discussion on GitHub](https://github.com/apollographql/router/discussions/2147)." />
```

Comment:
I didn't mean to leave this in, because my React skills weren't good enough to get `<ExperimentalFeature>` rendering nicely on my first try. Great if you've got it!

Reply:
This should do it: https://github.com/apollographql/docs/pull/614